### PR TITLE
(v8) Add guide for issuing app access certs for robots

### PR DESCRIPTION
**Pull request by r0mant**
Add a guide for issuing app access certificates for service accounts with tctl auth sign added in https://github.com/gravitational/teleport/pull/8717.

### CODE REVIEW

File: docs/config.json [hunk 1]
```diff
@@ -99,7 +99,8 @@
             { "title": "Integrating with JWT", "slug": "/application-access/guides/jwt/" },
             { "title": "API Access", "slug": "/application-access/guides/api-access/" },
             { "title": "AWS Console Access", "slug": "/application-access/guides/aws-console/" },
-            { "title": "Dynamic Registration", "slug": "/application-access/guides/dynamic-registration/" }
+            { "title": "Dynamic Registration", "slug": "/application-access/guides/dynamic-registration/" },
+            { "title": "Automation", "slug": "/application-access/guides/automation/" }
```
Comment (r0mant) [inline docs/config.json:103]: I wonder if something like `Service Accounts` would be a better name for this page, @klizhentas let me know what you think.

Comment (justinas) [reply docs/config.json:103]: I like the "service account" terminology, it is used in plenty of other software, e.g. K8s.

**Review by justinas — APPROVED** [review]

File: docs/config.json (hunk 1 above)
Comment (klizhentas) [reply docs/config.json:103]: I agree, service accounts is a well-understood term - better than robots

File: docs/pages/application-access/guides/automation.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  name: robot-impersonator
+spec:
+  allow:
+    impersonate:
+      users: ["robot"]
+      roles: ["robot"]
+```
+
+User `alice` can now impersonate the robot account and use its identity to
+issue short-lived certificates for accessing Grafana application.
```
Comment (klizhentas) [inline docs/pages/application-access/guides/automation.mdx:44]: 240 hours is not very short lived, let's limit to 24 hours?

File: docs/pages/application-access/guides/automation.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
+---
```
Comment (klizhentas) [inline docs/pages/application-access/guides/automation.mdx:1]: I feel like this article should wait for certbot to be completed. 240 hours for a non-expiring cert is a bit too much. I feel like we promote a bad security practice here.

**Review by klizhentas — CHANGES_REQUESTED** [review]

**Comment by zmb3** [comment]
Closing this one - we will write a guide for using machine ID with app access when we launch support for app access.

**Pull request closed without merging**